docs/root/operations/admin.rst

```diff
@@ -9,6 +9,25 @@ modify different aspects of the server:
 * :ref:`v1 API reference <config_admin_v1>`
 * :ref:`v2 API reference <envoy_api_msg_config.bootstrap.v2.Admin>`
 
+.. attention::
+
+  The administration interface in its current form both allows destructive operations to be
+  performed (e.g., shutting down the server) as well as potentially exposes private information
+  (e.g., stats, cluster names, cert info, etc.). It is **critical** that access to the
+  administration interface is only allowed via a secure network. This involves setting up
+  an appropriate firewall or optimally only allowing access to the administration listener via
+  localhost. This can be accomplished with a v2 configuration like the following:
+
+  .. code-block:: yaml
+
+    admin:
+      access_log_path: /tmp/admin_access.log
+      address:
+        socket_address: { address: 127.0.0.1, port_value: 9901 }
+
+  In the future additional security options will be added to the administration interface. This
+  work is tracked in `this <https://github.com/envoyproxy/envoy/issues/2763>`_ issue.
+
```

Comment:
Do you also want to point out that any host with access to the admin console should _only_ be on a trusted network, and not have access to an untrusted network. CSRF seems most dangerous in this mixed mode, where you have confused deputy.

Reply:
OK will clarify.